#!/usr/bin/env python
from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("p_gain", double_t, 0, "P Gain", 0.0, 0.0, 100.0)
gen.add("i_gain", double_t, 0, "I Gain", 0.0, 0.0, 100.0)
gen.add("d_gain", double_t, 0, "D Gain", 0.0, 0.0, 100.0)

exit(gen.generate(
    "auv_control", "z_axis_controller", "ZAxisController"))
